src/gedi_subset/subset.py

```diff
@@ -41,7 +41,7 @@ class CMRHost(str, Enum):
     nasa = "cmr.earthdata.nasa.gov"
 
 
-logicals = {"L4A": "10.3334/ORNLDAAC/2056", "L2A": "10.5067/GEDI/GEDI02_A.002"}
+logical_doi = {"L4A": "10.3334/ORNLDAAC/2056", "L2A": "10.5067/GEDI/GEDI02_A.002"}
```

Comment:
I'm getting nit-picky here. I suggested the plural form for this name (`logical_dois`) because it is a collection, not a single value.